### PR TITLE
Add feature toggle for flow SLA monitoring reactions

### PR DESCRIPTION
**Pull request by rozdy**

### CODE REVIEW

**Review by dpoltavets — APPROVED** [review]

**Review by niksv — APPROVED** [review]

File: docker/db-migration/migrations/006-add-flow-latency-monitoring-reactions-toggle.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      author: rozdy
+      changes:
+        - tagDatabase:
+            tag: 006-add-flow-latency-monitoring-reactions-toggle
+
+  - changeSet:
+      id: add_flow_latency_monitoring_reactions_toggle
+      author: rozdy
+      changes:
+        - sql: "CREATE PROPERTY config.flow_latency_monitoring_reactions IF NOT EXISTS LONG"
```
Comment (surabujin) [inline docker/db-migration/migrations/006-add-flow-latency-monitoring-reactions-toggle.yaml:13]: why LONG?

File: src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/service/ActionService.java [hunk 2]
```diff
@@ -142,7 +145,7 @@ public void saveFlowLatency(String flowId, String direction, long latency) {
     public void sendFlowSyncRequest(String flowId) {
         Flow flow = flowRepository.findById(flowId)
                 .orElseThrow(() -> new IllegalStateException(format("Flow %s not found.", flowId)));
-        if (LATENCY_BASED_STRATEGIES.contains(flow.getPathComputationStrategy())) {
+        if (LATENCY_BASED_STRATEGIES.contains(flow.getPathComputationStrategy()) && isReactionsEnabled()) {
```
Comment (surabujin) [inline src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/service/ActionService.java:148]: You still write in a log notification about future reaction, despite value of feature toggle.

File: docker/db-migration/migrations/006-add-flow-latency-monitoring-reactions-toggle.yaml (hunk 1 above)
Comment (rozdy) [reply docker/db-migration/migrations/006-add-flow-latency-monitoring-reactions-toggle.yaml]: Fixed.

File: src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/service/ActionService.java (hunk 2 above)
Comment (rozdy) [reply src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/service/ActionService.java:148]: Fixed.

**Review by surabujin — APPROVED** [review]

**Pull request merged**